src/containers/LoginContainer.jsx

```diff
@@ -117,7 +117,7 @@ Authentication process.'
               </div>
             </form>
           </div>
-
+           <Link to='https://e-comm-team-techsmith-bn-staging-fmp3.onrender.com/api/auth/google'>
```

Comment:
Include this in the .env, I mean the base url since we believe this might change in the future